playbooks/brick_reset.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Reset the bricks
+  remote_user: root
+  gather_facts: no
+  hosts: all
+  vars:
+     gluster_infra_reset_mnt_paths:
```

Comment:
can the vars be separated out to a separate file? In that case, we do not need to provide multiple playbooks - brick_reset.yml, brick_reset_vdo.yml etc ?